src/ui-lib/sass/05-components/01-atoms/06-buttons/_08-toggle-menu.scss

```diff
@@ -1,31 +1,105 @@
+$grav-sp-icon: 0.35rem;
+$grav-tr-icon-hover: 0.1rem;
+$grav-tr-icon-rotate: 135deg;
+$grav-tr-icon-scale: 1.2;
+$grav-size-icon: 33px;
+
 .grav-c-toggle-menu {
+  display: flex;
+  height: $grav-size-icon;
   margin-top: 0;
-  border: $grav-st-thickness solid $grav-co-neutral-asphalt;
+  border: hidden;
```

Comment:
that's only `border-style` that can be `hidden`, remember that `border` is just a shorthand property.